workflow/WorkflowService.go

```diff
@@ -306,12 +306,12 @@ func (service *WorkflowService) GetAdminStats() *piazza.JsonResponse {
 // GetEventType TODO
 func (service *WorkflowService) GetEventType(id piazza.Ident) *piazza.JsonResponse {
 
-	event, err := service.eventTypeDB.GetOne(piazza.Ident(id))
-	if err != nil {
-		return service.statusNotFound(id)
+	event, found, err := service.eventTypeDB.GetOne(piazza.Ident(id))
+	if !found {
+		return service.statusNotFound(err)
```

Comment:
Yes, in GetByID in index.go in pz-gocommon/elasticsearch if esi.ItemExists == false, then it returns a GetResult with Found = false and an error  that includes the es id, index, and type